raiden/ui/cli.py

```diff
@@ -112,12 +113,12 @@ def check_json_rpc(client):
         sys.exit(1)
     else:
         if client_version.startswith('Parity'):
-            print(
-                "Parity is not currently supported.\n"
-                "Waiting for the clients to expose the transaction's status code in the receipt.\n"
-                "https://github.com/ethereum/EIPs/pull/658"
-            )
-            sys.exit(1)
+            major, minor, patch = [
+                int(x) for x in re.search('//v(\d)\.(\d)\.(\d)', client_version).groups()
+            ]
+            if minor < 7 or (minor == 7 and patch < 6):
```

Comment:
i think this would be much clearer `if (major, minor, patch) < (1, 7, 6):`

Reply:
Hey thanks, you are right! Tuple comparison would make this check much clearer.